### DISCUSSION TITLE
Mod Addition: Mob Grinding Utils

### DISCUSSION THREAD

**Discussion by iconmaster5326** [NillerMedDild/Enigmatica6#4446] [Ideas]
https://www.curseforge.com/minecraft/mc-mods/mob-grinding-utils

This mod adds a bunch of cool things for making classical mob farms, not to mention the generally lovely absorption hopper and jumbo tank. I would reckon that the recipes would need to be made harder, so it's not always the first choice when making a mob farm, but I quite miss it from the pack.

**Comment by MuteTiefling** [comment]
Everything in that mod is already covered by other mods in the pack: https://wiki.enigmatica.net/enigmatica6/gameplay/what-can-do.../mobs-spawning-and-killing
